src/containers/WriteReviewPage.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+import React from 'react';
+import queryString from 'query-string';
+import scriptjs from 'scriptjs';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import '../components/powerreviews.less';
```

Comment:
If the powerreviews code was in a separate component, the styles would not have to be imported twice and the component can adjust depending on where it's displayed